production/technical_userguide.md

```diff
@@ -323,425 +338,170 @@ Below is an example of the format of the response body.
 
-**Headers**
+For ease of use, we have listed the `cURL` commands for the operations below. The job ID (`48`) will be different for your job.
 
-* `Content-Location: https://api.bcda.cms.gov/api/v1/jobs/44`
+**`cURL` command**
 
-#### 3. Check the status of the export job
+`curl -v https://api.bcda.cms.gov/api/v1/jobs/48 \
+-H 'Authorization: Bearer {token}'`
 
-**Request**
+##### 4. Retrieve the NDJSON output file
 
-`GET https://api.bcda.cms.gov/api/v1/jobs/44`
+Instructions for retrieving the output file have been detailed in previous sections. See [retrieving NDJSON output file(s)](https://bcda.cms.gov/production/technical-user-guide/#4-retrieve-ndjson-output-files){:target="_blank"} above.
 
-Using the *Content-Location* header value from the Patient resource type data export response, you can check the status of the export job. The status will change from `202 Accepted` to `200 OK` when the export job is complete and the data is ready to be downloaded.
+For ease of use, we have listed the `cURL` commands for the operation below. The job ID (`48`) and file name for the NDJSON file (`4e2cd98c-4746-4138-872b-24778c000b02.ndjson`) will be different for your job.
 
-**Headers**
+**`cURL` command**
 
-* `Authorization: Bearer {token}`
+`curl https://api.bcda.cms.gov/data/48/4e2cd98c-4746-4138-872b-24778c000b02.ndjson \
+-H 'Authorization: Bearer {token}'`
 
-**cURL Command**
+###### Retrieve the most current beneficiary claims data for existing beneficiaries AND historical data for newly attributed beneficiaries
 
-```
-curl -v https://api.bcda.cms.gov/api/v1/jobs/44 \
--H 'Authorization: Bearer {token}'
-```
+This section outlines the steps to make a request for ***both*** incremental claims data for existing beneficiaries and 7 years of historical data for beneficiaries that are newly attributed to your ACO within one API call.
 
-**Responses**
+You will need to make a request to the `/Group` endpoint using the `/Group/all` identifier as well as the `_since` parameter. The “`all`” identifier designates that you would like data for “all beneficiaries associated with your ACO”. At this time, `all` is the only group identifier that BCDA offers. 
 
-* `202 Accepted` indicates that the job is processing. Headers will include X-Progress: In Progress (The X-Progress header contains text indicating the job's status in BCDA's workflow. When the status is In Progress, an estimated progress percentage is also included.)
-* `200 OK` indicates that the job is complete.
+**Note: While most claims data is updated on a weekly cadence (and we encourage you to retrieve your ACO’s claims data weekly), ACO attribution is only updated one time per month, so you will only be able to retrieve historical claims data for newly attributed beneficiaries once per month. Additional calls will only yield duplicative data for these newly attributed beneficiaries.**
```

Comment:
This isn't showing up as bold on the site.
<img width="626" alt="Screen Shot 2020-08-01 at 4 19 11 PM" src="https://user-images.githubusercontent.com/37818548/89109839-c0f4f400-d412-11ea-95a9-e40371ba470b.png">